src/langchain_google_cloud_sql_pg/vectorstore.py

```diff
@@ -742,7 +742,7 @@ async def aapply_vector_index(
         params = "WITH " + index.index_options()
         function = index.distance_strategy.index_function
         name = name or index.name
-        stmt = f"CREATE INDEX {'CONCURRENTLY' if concurrently else ''} {name} ON {self.table_name} USING {index.index_type} ({self.embedding_column} {function}) {params} {filter};"
+        stmt = f"CREATE INDEX {'CONCURRENTLY' if concurrently else ''} {name} ON \"{self.table_name}\" USING {index.index_type} ({self.embedding_column} {function}) {params} {filter};"
```

Comment:
ditto here, would single quote be more readable?
```suggestion
        stmt = f"CREATE INDEX {'CONCURRENTLY' if concurrently else ''} {name} ON '{self.table_name}' USING {index.index_type} ({self.embedding_column} {function}) {params} {filter};"
```